src/content/learn/react-developer-tools.md

```diff
@@ -36,26 +37,25 @@ yarn global add react-devtools
 npm install -g react-devtools
 ```
 
-Next open the developer tools from the terminal:
+Daha sonra, terminalden React Developer Tools'u açın:
 ```bash
 react-devtools
 ```
 
-Then connect your website by adding the following `<script>` tag to the beginning of your website's `<head>`:
+Daha sonra, `<script>` etiketini web sitenizin `<head>` etiketinin başlangıcına etkleyerek bağlanın :
 ```html {3}
 <html>
   <head>
     <script src="http://localhost:8097"></script>
 ```
+Web sitenizi geliştirici araçlarında görüntülemek sayfayı yenileyin.
```

Comment:
it can be ?

`Web sitenizi geliştirici araçlarında görüntülemek için sayfayı yenileyin.`